deepmd/entrypoints/main.py

```diff
@@ -162,6 +163,14 @@ def parse_args(args: Optional[List[str]] = None):
         default="out.json",
         help="The output file of the parameters used in training.",
     )
+    parser_train.add_argument(
+        "-mt",
+        "--multi_task",
+        type=bool,
+        default=False,
```

Comment:
I suggest to use [`action="store_true"`](https://docs.python.org/3/library/argparse.html#action) instead.
```suggestion
        action="store_true",
```